build.sbt

```diff
@@ -4,8 +4,8 @@ version := "1.0"
 
 lazy val `play-webgoat` = (project in file(".")).enablePlugins(PlayScala)
 
-crossScalaVersions := Seq("2.12.7", "2.11.12")
-scalaVersion := "2.12.8"
+crossScalaVersions := Seq("2.12.8", "2.11.12")
+scalaVersion := crossScalaVersions.value.head // tc-skip
```

Comment:
This is a good solution for now. Whenever we have a cross-build, we can encode it as such and TemplateControl won't touch the scalaVersion line